diff -urN luminance-hdr-2.3.1/luminance-hdr.desktop luminance-hdr-2.3.1-patched/luminance-hdr.desktop
--- luminance-hdr-2.3.1/luminance-hdr.desktop	2013-04-08 05:41:14.000000000 +1100
+++ luminance-hdr-2.3.1-patched/luminance-hdr.desktop	2014-01-15 17:03:02.835756875 +1100
@@ -1,12 +1,11 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=Luminance HDR
 GenericName=HDR imaging tool
 GenericName[de]=Workflow für HDR-Bilder
 GenericName[it]=Workflow per immagini HDR
 GenericName[ru]=Работа с HDR-фотографией
 GenericName[tr]=HDR Resim Düzenleyici
-Categories=Graphics;RasterGraphics;Photography;Viewer;Qt;
+Categories=Graphics;2DGraphics;RasterGraphics;Photography;Viewer;Qt;
 Comment=Create and tonemap HDR images
 Comment[de]=Luminance HDR ist eine grafische Benutzeroberfläche zur Erzeugung von HDR-Bildern und deren Dynamikkompression
 Comment[en_US]=Luminance is a graphical user interface used to create and tonemap HDR images
