diff -urN luminance-hdr-2.2.0/luminance-hdr.desktop luminance-hdr-2.2.0-patched/luminance-hdr.desktop
--- luminance-hdr-2.2.0/luminance-hdr.desktop	2012-01-23 01:08:25.000000000 +1100
+++ luminance-hdr-2.2.0-patched/luminance-hdr.desktop	2012-01-31 19:25:16.161352785 +1100
@@ -1,12 +1,11 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=Luminance HDR
 GenericName=HDR imaging tool
 GenericName[de]=Workflow für HDR-Bilder
 GenericName[it]=Workflow per immagini HDR
 GenericName[ru]=Работа с HDR-фотографией
 GenericName[tr]=HDR Resim Düzenleyici
-Categories=Graphics;RasterGraphics;Photography;Viewer;Qt;
+Categories=Graphics;2DGraphics;RasterGraphics;Photography;Viewer;Qt;
 Comment=Create and tonemap HDR images
 Comment[de]=Luminance HDR ist eine grafische Benutzeroberfläche zur Erzeugung von HDR-Bildern und deren Dynamikkompression
 Comment[en_US]=Luminance is a graphical user interface used to create and tonemap HDR images
@@ -16,8 +15,6 @@
 Comment[tr]=HDR resim dosyaları oluşturun ve ton haritalamalarını düzenleyin
 Exec=luminance-hdr --gui %F
 Icon=luminance-hdr
-MimeType=
-Path=
 StartupNotify=true
 StartupWMClass=luminance-hdr
 Terminal=false
